coda/coda_validate/models.py

```diff
@@ -1,7 +1,36 @@
 from django.db import models
 from datetime import datetime
 
-# Create your models here.
+
+class ValidateManager(models.Manager):
+
+    def last_verified_status_counts(self):
+        results = self._verified_counts()
+        return VerifiedCountsResultFormatter(results).format()
+
+    def _verified_counts(self):
+        return (self.values('last_verified_status')
+                    .annotate(count=models.Count('last_verified_status')))
```

Comment:
Was this from running the tests, or did you discover this when you went to `/validate/check.json`?

Also, did you remove the existing container before you started the freshly built image?